src/renderers/native/__tests__/ReactNativeAttributePayload-test.js

```diff
@@ -15,227 +15,240 @@ var ReactNativePropRegistry = require('ReactNativePropRegistry');
 var diff = ReactNativeAttributePayload.diff;
 
 describe('ReactNativeAttributePayload', () => {
-
   it('should work with simple example', () => {
-    expect(diff(
-      {a: 1, c: 3},
-      {b: 2, c: 3},
-      {a: true, b: true}
-    )).toEqual({a: null, b: 2});
+    expect(diff({a: 1, c: 3}, {b: 2, c: 3}, {a: true, b: true})).toEqual({
+      a: null,
+      b: 2,
+    });
   });
 
   it('should skip fields that are equal', () => {
-    expect(diff(
-      {a: 1, b: 'two', c: true, d: false, e: undefined, f: 0},
-      {a: 1, b: 'two', c: true, d: false, e: undefined, f: 0},
-      {a: true, b: true, c: true, d: true, e: true, f: true}
-    )).toEqual(null);
+    expect(
+      diff(
+        {a: 1, b: 'two', c: true, d: false, e: undefined, f: 0},
+        {a: 1, b: 'two', c: true, d: false, e: undefined, f: 0},
+        {a: true, b: true, c: true, d: true, e: true, f: true},
+      ),
+    ).toEqual(null);
   });
 
   it('should remove fields', () => {
-    expect(diff(
-      {a: 1},
-      {},
-      {a: true}
-    )).toEqual({a: null});
+    expect(diff({a: 1}, {}, {a: true})).toEqual({a: null});
   });
 
   it('should remove fields that are set to undefined', () => {
-    expect(diff(
-      {a: 1},
-      {a: undefined},
-      {a: true}
-    )).toEqual({a: null});
+    expect(diff({a: 1}, {a: undefined}, {a: true})).toEqual({a: null});
   });
 
   it('should ignore invalid fields', () => {
-    expect(diff(
-      {a: 1},
-      {b: 2},
-      {}
-    )).toEqual(null);
+    expect(diff({a: 1}, {b: 2}, {})).toEqual(null);
   });
 
   it('should use the diff attribute', () => {
     var diffA = jest.fn((a, b) => true);
     var diffB = jest.fn((a, b) => false);
-    expect(diff(
-      {a: [1], b: [3]},
-      {a: [2], b: [4]},
-      {a: {diff: diffA}, b: {diff: diffB}}
-    )).toEqual({a: [2]});
+    expect(
+      diff(
+        {a: [1], b: [3]},
+        {a: [2], b: [4]},
+        {a: {diff: diffA}, b: {diff: diffB}},
+      ),
+    ).toEqual({
+      a: [2],
+    });
```

Comment:
This is not expected that the toEqual argument is split on multiple lines.

I cannot repro neither on the website nor master: https://prettier.github.io/prettier/#%7B%22content%22%3A%22function%20f()%20%7B%5Cn%20%20function%20f()%20%7B%5Cn%20%20%20%20expect(diff(%5Cn%20%20%20%20%20%20%7Ba%3A%20%5B1%5D%2C%20b%3A%20%5B3%5D%7D%2C%5Cn%20%20%20%20%20%20%7Ba%3A%20%5B2%5D%2C%20b%3A%20%5B4%5D%7D%2C%5Cn%20%20%20%20%20%20%7Ba%3A%20%7Bdiff%3A%20diffA%7D%2C%20b%3A%20%7Bdiff%3A%20diffB%7D%7D%2C%5Cn%20%20%20%20)).toEqual(%7Ba%3A%20%5B2%5D%7D)%3B%5Cn%20%20%7D%5Cn%7D%5Cn%22%2C%22options%22%3A%7B%22printWidth%22%3A80%2C%22tabWidth%22%3A2%2C%22singleQuote%22%3Atrue%2C%22trailingComma%22%3A%22all%22%2C%22bracketSpacing%22%3Atrue%2C%22jsxBracketSameLine%22%3Atrue%2C%22parser%22%3A%22babylon%22%2C%22doc%22%3Afalse%7D%7D